lib/debug/breakpoint.rb

```diff
@@ -348,15 +348,19 @@ def try_enable quiet: false
       rescue ArgumentError => e
         raise if retried
         retried = true
+        sig_method_name = @sig_method_name
```

Comment:
Instance variable like `@sig_method_name` is `nil` inside `module_eval`. We need to assign it to a local for accessing it in `module_eval`.